python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -274,18 +279,27 @@ def render_egoposes_on_fancy_map(self,
                                      nusc: NuScenes,
                                      scene_tokens: List = None,
                                      verbose: bool = True,
-                                     out_path: str = None) -> np.ndarray:
+                                     out_path: str = None,
+                                     render_egoposes: bool = True,
+                                     render_egoposes_range: bool = True,
+                                     render_legend: bool = True) -> np.ndarray:
         """
         Renders each ego pose of a list of scenes on the map (around 40 poses per scene).
         This method is heavily inspired by NuScenes.render_egoposes_on_map(), but uses the map expansion pack maps.
         :param nusc: The NuScenes instance to load the ego poses from.
         :param scene_tokens: Optional list of scene tokens corresponding to the current map location.
         :param verbose: Whether to show status messages and progress bar.
         :param out_path: Optional path to save the rendered figure to disk.
+        :param render_egoposes: Whether to render ego poses.
+        :param render_legend: Whether to render road segment render_legend.
```

Comment:
same here, also the description seem confusing should it be legend not render_legend.